.github/workflows/reusable-conditional-skip.yml

```diff
@@ -12,13 +12,58 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
No updates needed to triggers in calling workflows, as they were already split on PR and protected branch push.